.circleci/config.yml

```diff
@@ -134,6 +134,18 @@ jobs:
       # Run linter
       - run: yarn lint
 
+  # Check if all files respect the prettier rules
+  run-prettier:
+    <<: *defaults_js
+
+    steps:
+      - checkout
+      - *restore_node_cache
+      - *install_node_modules
+
+      # Run linter
+      - run: yarn prettier --check "ts/**/*.ts*"
```

Comment:
This PR LGTM, what do you think about adding this to the scripts section of `package.json` file and run here the script defined? Can be useful to run locally the check in case of errors

Reply:
Sure, added in 0b6ce31a7c91091165ced5f6d31db1b0d4d63126!